app/assets/javascripts/discourse/components/topic-timer-info.js.es6

```diff
@@ -18,6 +19,9 @@ export default Ember.Component.extend(
     buildBuffer(buffer) {
       if (!this.get("executeAt")) return;
 
+      const topicClosed = this.get("topicClosed");
+      if (topicClosed !== undefined && (topicClosed ? "close" : "open") === this.get("statusType")) return;
```

Comment:
`topicClosed !== undefined` checks if the property is passed in. (Let me know if there is a more Ember-y way)

This is important because we don't want to assume the topic's status (closed vs open) and hide the topic timer information by mistake.